src/citrine/informatics/descriptors.py

```diff
@@ -89,9 +91,15 @@ def __eq__(self, other):
         except AttributeError:
             return False
 
-    def __init__(self, key: str, threshold: float = 1.0):
+    def __init__(self, key: str):
         self.key: str = key
-        self.threshold = threshold
+
+
+def InorganicDescriptor(key: str, threshold: Optional[float] = 1.0):
+    """[DEPRECATED] Use ChemicalFormulaDescriptor instead."""
+    logger.warning("InorganicDescriptor is deprecated and will soon be removed. "
```

Comment:
I believe the standard pattern for deprecations is to use the warnings package, e.g.:
https://github.com/CitrineInformatics/gemd-python/blob/master/taurus/__init__.py#L4
which was pattern-copied from a deprecated python standard module.

Reply:
changed